contents/docs/advanced/proxy.mdx

```diff
@@ -34,6 +39,12 @@ Options for deploying a reverse proxy include:
 - [Nuxt](/docs/advanced/proxy/nuxt)
 - [Pomerium](/docs/advanced/proxy/pomerium)
 
+## Best practices
+
+- We require that the proxy sets the `Host` header to the same host it's calling. Check the guides above on how to do that for several proxies.
+- Don't use a subdomain that includes `posthog`, `analytics`, `tracking`, or other similar words which might cause events to be blocked.
+- Make sure to [pass the proper `ui_host` parameter](/docs/libraries/js/config) when initializing PostHog, so that links to PostHog point to the correct host. This is required for tasks like authenticating the toolbar.
```

Comment:
```suggestion
- Make sure to [pass the proper `ui_host` parameter](/docs/libraries/js/config) when initializing PostHog, so that links to PostHog point to the correct host (like `us.posthog.com`). This is required for tasks like authenticating the toolbar.
```